lib/wordpress.js

```diff
@@ -44,7 +44,11 @@ function Client( settings ) {
 		// via settings.host
 		headers: {
 			Host: parsedUrl.host
-		}
+		},
+		basic_auth: settings.basicAuth ? {
+			user: settings.basicAuth.username,
+			pass: settings.basicAuth.password
+		} : null,
```

Comment:
Please either break before the operators:

```js
		basic_auth: settings.basicAuth
			? {
				user: settings.basicAuth.username,
				pass: settings.basicAuth.password
			}
			: null
```

Or invert the conditional:

```js
		basic_auth: !settings.basicAuth ? null : {
			user: settings.basicAuth.username,
			pass: settings.basicAuth.password
		}
```

And drop the trailing comma.